src/components/splash/index.js

```diff
@@ -27,19 +106,28 @@ class Splash extends React.Component {
             <h1 style={{textAlign: "center", marginTop: "-10px", fontSize: "29px"}}> Real-time tracking of virus evolution </h1>
           </div>
           {/* First: either display the error message or the intro-paragraph */}
-          {this.props.errorMessage ? (
+          {this.props.errorMessage || this.state.errorMessage ? (
             <CenterContent>
-              {displayError(this.props.errorMessage)}
+              <div>
+                <p style={{color: "rgb(222, 60, 38)", fontWeight: 600, fontSize: "24px"}}>
+                  {"😱 404, or an error has occured 😱"}
+                </p>
+                <p style={{color: "rgb(222, 60, 38)", fontWeight: 400, fontSize: "18px"}}>
+                  {`Details: ${this.props.errorMessage || this.state.errorMessage}`}
+                </p>
+                <p style={{fontSize: "16px"}}>
+                  {"If this keeps happening, or you believe this is a bug, please "}
+                  <a href={"mailto:hello@nextstrain.org"}>{"get in contact with us."}</a>
```

Comment:
I think @trvrb wanted to avoid real `mailto:` links for spam reasons?

Reply:
What's your understanding of the potential for spam? If you can see a way of keeping the link active without `mailto:` then please go for it. I don't like how on nextstrain.org they aren't links.